prow/plugins/config_test.go

```diff
@@ -105,8 +105,8 @@ func TestSetDefault_Maps(t *testing.T) {
 		{
 			name: "nothing",
 			expected: map[string]ConfigMapSpec{
-				"config/prow/config.yaml":  {Name: "config", Namespaces: []string{""}},
-				"config/prow/plugins.yaml": {Name: "plugins", Namespaces: []string{""}},
+				"config/prow/config.yaml":  {Name: "config", Namespaces: []string{""}, Clusters: []string{""}},
```

Comment:
I'm surprised to see an empty string here -- a user would have to explicitly set

```
cluster: ""
```

to get his behavior, right? What happens if we just pass an empty or nil slice?

Reply:
Those two fields `Namespaces` and `Clusters` are not set up by the user (namespace/additional_namespaces and cluster/additional_clusters are), I think:
https://github.com/kubernetes/test-infra/blob/f204f80c69e8e9c2d973dac6b57c6f0d0d2f4a61/prow/plugins/config.go#L411-L431

Instead, they are calculated by:

https://github.com/kubernetes/test-infra/blob/f204f80c69e8e9c2d973dac6b57c6f0d0d2f4a61/prow/plugins/config.go#L809-L813

Tested with nil/empty_slice ... not working ... the reason is above.